src/yio-interface/integrationinterface.h

```diff
@@ -53,6 +53,20 @@ class IntegrationInterface {
     virtual bool addAvailableEntity(const QString& entityId, const QString& type, const QString& integration,
                                     const QString& friendlyName, const QStringList& supportedFeatures) = 0;
 
+    /**
+     * @brief addAvailableEntity
+     * @param entityId
+     * @param type
+     * @param integration
+     * @param friendlyName
+     * @param supportedFeatures
+     * @param customFeatures
+     */
```

Comment:
Either document method or remove empty template.
Yes, there are other empty doc templates in this class which need to be fixed... But new code should be better ;-)